bookwyrm/templates/manifest.json

```diff
@@ -0,0 +1,14 @@
+{% load static %}
+{
+  "name": "Bookwyrm",
```

Comment:
This should use `site.name` and `site.description` instead of the defaults.

Reply:
Fixed.